src/test/java/core/basesyntax/dao/FileDataReaderImplTest.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class FileDataReaderImplTest {
+    private final String inputFile = new String("src/main/resources/data.csv");
```

Comment:
1. Same goes for files that you use in tests, let's put them into this folder: src/test/resources/[your-files.csv]
2. why using `new String()`?